niworkflows/anat/ants.py

```diff
@@ -340,47 +323,37 @@ def init_brain_extraction_wf(
     # fmt: off
     wf.connect([
         (inputnode, trunc, [("in_files", "op1")]),
-        (inputnode, copy_xform, [(("in_files", _pop), "hdr_file")]),
         (inputnode, inu_n4_final, [("in_files", "input_image")]),
         (inputnode, init_aff, [("in_mask", "fixed_image_mask")]),
         (inputnode, norm, [("in_mask", fixed_mask_trait)]),
         (inputnode, map_brainmask, [(("in_files", _pop), "reference_image")]),
         (trunc, inu_n4, [("output_image", "input_image")]),
-        (inu_n4, res_target, [(("output_image", _pop), "input_image")]),
-        (res_tmpl, init_aff, [("output_image", "fixed_image")]),
-        (res_target, init_aff, [("output_image", "moving_image")]),
+        (inu_n4, res_target, [(("output_image", _pop), "in_file")]),
+        (res_tmpl, init_aff, [("out_file", "fixed_image")]),
+        (res_target, init_aff, [("out_file", "moving_image")]),
         (init_aff, norm, [("output_transform", "initial_moving_transform")]),
         (norm, map_brainmask, [
             ("reverse_transforms", "transforms"),
             ("reverse_invert_flags", "invert_transform_flags"),
         ]),
         (map_brainmask, thr_brainmask, [("output_image", "input_image")]),
-        (thr_brainmask, dil_brainmask, [("output_image", "op1")]),
-        (dil_brainmask, get_brainmask, [("output_image", "op1")]),
+        (map_brainmask, inu_n4_final, [("output_image", "weight_image")]),
```

Comment:
Having trouble tracing how this originally connected through...

Reply:
Although not ideal, this might actually help -> https://1942-74174878-gh.circle-artifacts.com/0/docs/_build/no_version_html/html/api/niworkflows.anat.ants.html#niworkflows.anat.ants.init_brain_extraction_wf

Reply:
Okay, so it looks like we're no longer using Atropos to select white matter at all. What purpose does it now serve?

Reply:
Yes, we still use ATROPOS:
- To refine the binary mask (as originally done by antsBrainExtraction.sh)
- To obtain the WM posterior (i.e., select the WM).

In some sense, now we use it even more :D 

However, it is a bit implicit - see the `disconnect()` call below, to replace the final outputs of the brain-extraction workflow with those coming from the ATROPOS workflow.